### ISSUE TITLE
some problems on training models

### ISSUE THREAD

**Issue by Improving00252** [AolinFeng/DMP-HEVC-ICME2021#1]
Hi , i am a college student and just read your paper , i am amazed by your ideas of replacing the time-comsuming RDO with CNN to predict the depth-map ! Then i choose this one as the theme of my oral presentation . 

Due to the course need , we`re asked to implement the work we present , and add some of our ideas into it . 
After i took the online Pytorch Maching Learning Course on youtube , there is still some problems that i can`t figure it out 

Most of the implement on youtube course were train with optimizer and the loss to calculate the loss , in the paper i learned that you idea of using the "multi loss" bring a good performance , and i didn`t find it in the code ... , so  might need to seek for  you help on how to designed a new model that trained with a new dataset .

Thank you for the great work and look forward to your reply
                                                                                                                                                                                                          --York

**Comment by AolinFeng** [comment]
The loss function can be constructed according to the equations of section 2.4 in the paper. Good luck.

**Comment by Improving00252** [comment]
Appreciate your reply !
I'm now working on it ,  hope to finish it soon !

--York

Aolin Feng ***@***.***> 於 2022年12月24日 週六 下午1:05 寫道：
